lib/build_pack_utils/process.py

```diff
@@ -43,12 +43,7 @@
 
 def _enqueue_output(proc, queue):
     if not proc.quiet:
-        for line in iter(proc.stdout.readline, b''):
-            try:
-                line = line.decode('utf-8')
-            except UnicodeDecodeError as e:
-                queue.put((proc, e))
-                continue
+        for line in iter(proc.stdout.readline, ''):
```

Comment:
We no longer do an explicit `decode` here, since `proc.stdout` is returning strings (thanks to a `text=True` somewhere). That decode is happening implicitly somewhere, so there might be an uncaught decode exception here. That said, since the buildpack runs in controlled environments (CF stacks), I'm skeptical anyone would actually run into that case, in practice.